README.md

```diff
@@ -27,11 +27,24 @@ Then run `sh openshift/setup-robots.sh`, which will create a new project named
 `robots` containing an application corresponding to each of the three
 applications within this project.
 
-Then deploy Cryostat to the namespace. The easiest way to achieve this is
-to clone the [operator](https://github.com/cryostatio/cryostat-operator)
-and run `make deploy` (no build is required - an image from the Quay.io remote
-registry will be pulled by your CRC node and deployed).
+Then deploy Cryostat to the namespace. The easiest way to do this is to open
+the OpenShift Console and navigate to OperatorHub. Search for *Cryostat* and
+install the *Cryostat Operator* in your *robots* project. Then create a
+Cryostat resource to instruct the Operator to create an actual deployment.
```

Comment:
Might be worth mentioning to install cert-manager in this case as well. (https://cert-manager.io/docs/installation/)

Reply:
Oh right. Good catch.